Python/correlation/test/test_correlation.py

```diff
@@ -0,0 +1,142 @@
+"""
+Provides test to visualize results, and check how accurate the result is when varying
+sample frequency, pulse length, position of hydrophones and pinger, and window length.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+import Python.correlation.correlation as correl
+import Python.correlation.tdoa as tdoa
+
+import Python.signal_generation.noise as sg_noise
+import Python.signal_generation.positioning as sg_pos
+import Python.signal_generation.receiver as sg_rec
+import Python.signal_generation.source as sg_src
+
+#@pytest.mark.plot
```

Comment:
```suggestion
@pytest.mark.plot
```
Can then be run with by adding `--runplot`, so `pytest --runplot <path>`